.obelisk/impl/github.json

```diff
@@ -1,8 +1,8 @@
 {
   "owner": "obsidiansystems",
   "repo": "obelisk",
-  "branch": "develop",
+  "branch": "pzp",
```

Comment:
?

Reply:
Uh, weird. I did this change from inside pzp's project, but I don't remember even unpacking that thunk.